js/gettingStarted.js

```diff
@@ -0,0 +1,34 @@
+let os = detectOS()
+window.onload = init
```

Comment:
Why do we check for `window.onload` instead of `document.ready` in `jQuery`?

Reply:
I changed this